### ISSUE TITLE
Accessibility: MAS40B: Inspect: Name property is empty for the "Ext Install Python" read only text

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#54]
#A11yMAS
#MAS40B
#Wipro
#vscode-website

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Version-10.0.14393 Build 14393.607
Browser-Edge
Edge Version-Microsoft Edge 38.14393.607.0

**Repro Steps:**
1. Open https://marketplace.visualstudio.com/items?itemName=donjayamanne.python in EDGE
2. Navigate to the "Ext Install Python" read only text available under "Installation" 
3. Observe the "Name" property in Inspect

**Actual Result:**
The "Name" property is empty
**_Narrator Behaviour:_**
Narrator announces as "Read Only Ext Install Python"

**Expected Result:**
The name property should be specific so that the users are able to understand the exact purpose of the read only text
**_Eg:_**
"Ext Install Python" Read only Python installation Command

**User Impact:**
Visually impaired users  will not get to know the purpose of the command text as the screen reader will not announce anything when the user arrives at the installation command
Providing the name property enables the compatibility with the Screen Reader, which may be used by people with disabilites.

**Screenshot:**
![name property is empty for read only python install command](https://cloud.githubusercontent.com/assets/24825441/21758769/a7b845be-d664-11e6-8e47-0d1383fd6bd2.png)

**Comment by waderyan** [comment]
Marketplace issue.
